internal/oprf/oprf.go

```diff
@@ -37,18 +36,16 @@ func NewOPRF(m, baseOT int) (OPRF, error) {
 // q stores the received OT extension matrix chosen with secret
 // seed s.
 type Key struct {
-	block cipher.Block
-	s     []byte   // secret choice bits
-	q     [][]byte // m x k bit matrice
+	s []byte   // secret choice bits
+	q [][]byte // m x k bit matrice
 }
 
 // Encode computes and returns OPRF(k, in)
-func (k Key) Encode(j uint64, in []byte, hIdx uint8) (out []byte, err error) {
-	// compute q_i ^ (C(r) & s)
-	out = crypto.PseudorandomCodeWithHashIndex(k.block, in, hIdx)
+func (k Key) Encode(j uint64, pseudorandomBytes []byte) (out []byte, err error) {
+	out = pseudorandomBytes
```

Comment:
you can just change `pseudorandomBytes` without declaring `out`, we won't need that value anymore once it is encoded.

Reply:
Okay, done. It speeds up each encode by 2 ns